specification/definitions/Address.json

```diff
@@ -1,20 +1,17 @@
 {
   "type": "object",
   "required": [
-    "street",
-    "number",
-    "postal_code",
+    "addressline_1",
     "city",
-    "cc"
+    "country_code"
   ],
   "properties": {
-    "street": {
+    "addressline_1": {
       "type": "string",
-      "example": "Hoofdweg"
+      "example": "Hoofdweg 679"
     },
-    "number": {
-      "type": "string",
-      "example": "679"
+    "addressline_2": {
```

Comment:
Dat kan, maar het praat misschien wat makkelijker als we alles in develop hebben en dan in 1 keer alle endpoints gaan bespreken, zonder dat we steeds van branch moeten switchen.